### ISSUE TITLE
Animations / highlights do not change until after swiping (with react-nagivation)

### ISSUE THREAD

**Issue by sophia-2b** [timomeh/react-native-material-bottom-navigation#85]
### What kind of Issue is this?

- () Bug Report
- (X) Question / Problem
- () Discussion / Feature Request

### How are you using the Bottom Navigation?

- (X) I use the Bottom Navigation together with react-navigation; the Issue is not appearing when I'm using react-navigation's TabBar instead.
- () I use the standalone version.

### Expected behavior
Animations and active colors to change while swiping.

### Actual behavior

Animations and active colors do not change until after the swiping animation is complete.

### Additional description and resources
I seem to be experiencing what was reported a long time ago here: [Animation after TabPress seems to have too much delay #7](https://github.com/timomeh/react-native-material-bottom-navigation/issues/7). However, pressing a tab results in the animation and active colors to update immediately. Swiping, however, has the delay noted there - nothing changes until after the swipe is complete.

Are there any configuration options or something I've overlooked to get the navigation to update upon swiping, rather than nothing changing until after swiping?

### Environment

- **React Native versions:** 0.55.2
- **react-native-material-bottom-navigation version:** 0.9.0
- **react-navigation version:** 1.5.11
- **I tested this with:** Android

**Comment by timomeh** [comment]
Technically, the Bottom Navigation gets the active Tab from react-navigation, and it seems like react-navigation updates the active tab not before the swipe is complete.
Have you swapped out the `NavigationComponent` with react-navigation's `TabBarBottom` and checked if it works like you expected? If their TabBarBottom works like you expected, then they implemented something which handles that, which I didn't implement.

**Comment by sophia-2b** [comment]
I tested swapping in the `TabBarTop` and the animation was during the swipe as expected. I can try using their `TabBarBottom` later today.

**Comment by sophia-2b** [comment]
Meh, tested real quick before I leave for work. Confirmed the animation is during swipe as well with the `TabBarBottom`

**Comment by timomeh** [comment]
Thank you very much for testing this out! If I understood you correctly, the `TabBarBottom` also is only active after the swipe is finished? In that case, this would be a feature request for react-navigation. Unfortunately I can't work around this. 😕

**Comment by sophia-2b** [comment]
Oh no sorry - `TabBarTop `and `TabBarBottom` both show the animation during the swipe. They sort of fade-in / fade-out the color and movements as you swipe. `NavigationComponent` will show the color fadein/out and animation only after the swiping of screens is completed.

**Comment by timomeh** [comment]
Ah, okay, I see. The `NavigationComponent` (respectively the material-bottom-navigation as of v0.9) has no ability to fade-in/out, so I can't simply patch it.

The next version of the material-bottom-navigation won't include out-of-the-box support for react-navigation though, because react-navigation implemented its own Bottom Navigation. As of now I haven't decided if I want to continue support for react-navigation.

I'll keep this issue open until I know more.

**Comment by timomeh** [comment]
I'll put this on hold since v1.0.0 doesn't contain out-of-the-box support for React Navigation anymore. Read more about the details [here](https://github.com/timomeh/react-native-material-bottom-navigation#react-navigation-support).

**Comment by timomeh** [comment]
I'm going to close this issue since I deprecated/removed out-of-the-box support for React Navigation.
